src/mpyl/steps/models.py

```diff
@@ -87,7 +87,7 @@ def from_configuration(run_properties: Dict, config: Dict):
 
         versioning = VersioningProperties(revision=versioning_config['revision'],
                                           branch=versioning_config['branch'],
-                                          pr_number=int(pr_num) if pr_num.isdigit() else None,
```

Comment:
I thought this was needed because if the pr num env var isn't set, it's parsed as the string 'N/A'.. Is that changed now?

Reply:
Yes it is, the problem was that in the monorepo, the default `pyaml_env` was used instead of our own
```from mpyl.utilities.pyaml_env import parse_config```
Ours replaces `N/A` with `None` so we can have optional config properties.

Reply:
Ah nice, good to know!